README.md

```diff
@@ -170,7 +170,7 @@ The following are some limitations with otto:
 
 * `use strict` will parse, but does nothing.
 * The regular expression engine ([re2/regexp](https://pkg.go.dev/regexp)) is not fully compatible with the ECMA5 specification.
-* Otto targets ES5. Some ES6 features e.g. Typed Arrays are not supported, PR's to add functionality are always welcome.
+* Otto targets ES5. Some ES6 features, e.g., Typed Arrays, are not supported. Pull requests to add functionality are always welcome.
```

Comment:
suggestion: no change needed. Omitting the comma is fine, only considered more correct in **American** English